src/components/EditorTopBar.tsx

```diff
@@ -26,24 +26,24 @@ export const EditorTopBar = ({
               <div className="dp-content-menu">
                 <ul className="dp-list-exit" id="dp-exit-editor">
                   <li role="none">
-                    <button type="button" role="menuitem">
+                    <a href="#" role="menuitem">
```

Comment:
El CI está fallando porque `#` no es un _href_ válido:

![image](https://user-images.githubusercontent.com/1157864/159032681-154df41d-0511-4a6f-86a5-8cb126435e10.png)

Una opción sería agregar dependencia con la configuración y luego (por ahora) usar la URL de login acá:

```diff
 export const EditorTopBar = ({
   onSave,
   title,
   ...otherProps
 }: EditorTopBarProps) => {
+  const { appConfiguration: { loginPageUrl } } = useAppServices();
   return (
     <div className="editor-top-bar" {...otherProps}>
```

```suggestion
                    <a href="{loginPageUrl}" role="menuitem">
```

Reply:
Con eso funciono de 10 @andresmoschini !!! 🚀